lib/img.js

```diff
@@ -42,15 +47,15 @@ module.exports = function img(source, context) {
           });
         }
         if (!source.svgAsImage) {
-          await imgSVG(source, context, svgo);
+          imgSVG(source, context, svgo);
           return resolve();
         }
 
         source.tag = 'img';
         // Strip xml tag
         source.content = source.fileContent.replace(RE_XML_TAG, '');
         if (source.compress) {
-          const result = await svgo.optimize(source.content);
+          const result = svgo.optimize(source.content);
```

Comment:
`await` is required here since call is async

Reply:
I removed the `await` due to a git pre-hook lint check failure.
Can you please have a look into the lint check?

Reply:
there's no longer a lint error after I refactored the method as `async`. Merge changes from `master` branch